README.md

```diff
@@ -97,7 +97,8 @@ Read [this](https://github.com/golang/go/wiki/Modules#how-to-install-and-activat
 
 ### Setting Up Local Dev Environment
 
-Building is supported on OS X, Linux, FreeBSD, and Windows.
+Building is supported on OS X, Linux, FreeBSD, and Windows
+In the case of Linux, GLIBC 2.34+ is required.
```

Comment:
More specifically we support

- Ubuntu 22.04 and newer
- MacOS 13 (Intel) and MacOS 14 (Arm) and newer
- Windows 10 and newer
- FreeBSD is not officially supported